source/variables/custom-selectors.css

```diff
@@ -1 +1,8 @@
-@custom-selector :--enter :hover, :active;
\ No newline at end of file
+/* 
+	Allows you to create your own selectors
+	http://cssnext.io/features/#custom-selectors
+	:--enter {
+	  ... styles ...
+	}	
+*/
+@custom-selector :--enter :hover, :focus, :active;
```

Comment:
I think you and Dan need to have a conversation about this, he asked why focus was not included.
I will remove it again but the two of you please connect.

Thanks!